apps/api/src/app/integrations/usecases/get-novu-integration/get-novu-integration.usecase.ts

```diff
@@ -0,0 +1,85 @@
+import { Injectable } from '@nestjs/common';
+import { IntegrationEntity, MessageRepository, IntegrationRepository } from '@novu/dal';
+import { GetNovuIntegrationCommand } from './get-novu-integration.command';
+import { ChannelTypeEnum, EmailProviderIdEnum } from '@novu/shared';
+import { startOfMonth, endOfMonth } from 'date-fns';
+
+@Injectable()
+export class GetNovuIntegration {
+  constructor(private messageRepository: MessageRepository, private integrationRepository: IntegrationRepository) {}
+
+  async execute(command: GetNovuIntegrationCommand): Promise<IntegrationEntity[]> {
```

Comment:
A small question that just seen, regarding the signature here, Why are we returning array and not an object?

Reply:
because GetDecryptedIntegrations also returns an array...